libs/sdk-ui-tests/stories/visual-regression/visualizations/insightStories.tsx

```diff
@@ -169,7 +169,7 @@ function plugVizStory(insight: IInsight, testScenario: IScenario<any>) {
         });
 
     const wrapper = testScenario.tags.includes("themed")
-        ? (child: any) => screenshotWrapper(wrapWithTheme(child)) // since themes are global anyway, wrap only once
+        ? (child: any) => screenshotWrapper(wrapWithTheme(child, testScenario.tags)) // since themes are global anyway, wrap only once
         : (child: any) => screenshotWrapper(child);
 
     const effectiveTheme = testScenario.tags.includes("themed") ? theme : undefined;
```

Comment:
You probably need to change this too so that the effective theme is the one with only the font if tags contain "font", right?